src/installer/tests/AppHost.Bundle.Tests/AppLaunch.cs

```diff
@@ -176,8 +178,7 @@ public void FrameworkDependent_NoBundleEntryPoint()
             }
         }
 
-        [Fact]
-        [PlatformSpecific(TestPlatforms.Windows)] // GUI app host is only supported on Windows.
+        [PlatformSpecificFact(TestPlatforms.Windows)] // GUI app host is only supported on Windows.
```

Comment:
Would it be better for this to be a regular ConditionalFact with a custom condition? We have thousands of ConditionalFact and ConditionalTheories in this repo,

`[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsWindows))]`

Or is there something special about Platforms that makes it worth it to create dedicated Fact/Theory attributes for them?

Reply:
Looks like this isn't yet available in Arcade: https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.XUnitExtensions.Shared/Attributes/ConditionalFactAttribute.cs

I'm not opposed to implementing it, but I'd rather not wait for the whole Arcade loop to make changes.

Reply:
I have submitted https://github.com/dotnet/arcade/pull/16190 to Arcade. I do not see anything preventing this from getting it enabled. see some discussion about nice to have cleanup in https://github.com/dotnet/arcade/pull/15668 but that looks like nice-to-have cleanup.